src/CopyableInput/CopyableInput.tsx

```diff
@@ -93,13 +93,18 @@ export class CopyableInput extends React.Component<Props, State> {
         />
         <div className="CopyableInput--links">
           {this.props.enableShow && (
-            <button type="button" className="CopyableInput--link" onClick={this.toggleHidden}>
+            <button
+              type="button"
+              className="CopyableInput--link"
+              onClick={this.toggleHidden}
+              aria-live="polite"
+            >
               {this.state.hidden ? "Show" : "Hide"}
```

Comment:
It looks like in Lockbox's version we also add accessibility tags to the show/hide and copied/copy text eg https://github.com/Clever/launchpad/pull/2988/files/c97eae2fba8400565e539b9d51c29fbccf30e59d#diff-866ea003e256b80e70105ef86c9bcee1R42

Reply:
When I tested earlier, those actually seemed detrimental, as I found that adding them in seemed to make VoiceOver unable to read the current text of the button when navigating to it (via CTRL+OPT+Rarrow), though of course I'm happy to add it in if I missed something.

As-is, I found that VoiceOver correctly noted the text of the button as it changed. What does adding the additional ones (per Lockbox's version) do here?

Reply:
Gotcha, thanks for testing that! I think in that case we can go with what you have. I'm not aware of specific benefits of what was done in Lockbox, it's very possible we just thought both were necessary (we're definitely still figuring out accessibility best practices)

Reply:
As discussed, I went ahead and catalogued the exact behavior of how VoiceOver reads it for me, where "with" is with the additional tags that the Lockbox version has, and "without" is the version I have in my commit for CopyableInput. The left (of semicolon) string is what VoiceOver read out upon focusing on the button, and the right (of semicolon) string is what VoiceOver read out upon clicking the button. Parentheses denote that it was only read out _before_ clicking the button.

With additional tags:
button ; (Copy,) button
button ; press (Copy) button

Without additional tags:
Show/Hide, button ; press Hide/Show button, Show/Hide
Copy/Copied! button ; press Copy/Copied! button

I am definitely no expert in accessibility and have never thoroughly researched how these tags work or how best to deploy them, so I've just gone off of reading documentation for the tags and trying out VoiceOver myself and seeing what gives the clearest, largest amount of info!